jazz_scm-webhook/config/stg-config.json

```diff
@@ -0,0 +1,64 @@
+{
+  "events_url": "https://{conf-apikey}.execute-api.{conf-region}.amazonaws.com/stg/jazz/events",
+  "event_status": {
+    "completed": "COMPLETED"
+  },
+  "event_handler": {
+    "bitbucket": "BITBUCKET"
```

Comment:
Key name should be SCM? Also this should be replaced as the service is getting built.

Reply:
This will change in next iteration when there is GitLabs added.